test/elixir/hologram/ex_js_consistency/erlang/binary_test.exs

```diff
@@ -0,0 +1,297 @@
+defmodule Hologram.ExJsConsistency.Erlang.BinaryTest do
+  @moduledoc """
+  IMPORTANT!
+  Each Elixir consistency test has a related JavaScript test in test/javascript/erlang/binary_test.mjs
+  Always update both together.
+  """
+
+  use Hologram.Test.BasicCase, async: true
+
+  @moduletag :consistency
+
+  describe "at/2" do
+    test "returns byte at position 0" do
+      assert :binary.at(<<5, 19, 72, 33>>, 0) == 5
+    end
+
+    test "returns byte at position 1" do
+      assert :binary.at(<<5, 19, 72, 33>>, 1) == 19
+    end
+
+    test "raises ArgumentError when position is out of range" do
+      assert_error ArgumentError, "argument error", fn ->
```

Comment:
I don't understand why the error only contains `"argument error"` as text.
Whereas other `ArgumentError`s below have a message such as  `"is a bitstring (expected a binary)"` or `"out of range"`.

Reply:
Interesting.

But then why even bother to construct a nice error in `binary.mjs`, if it's silently ignored anyway?


```js
      Interpreter.raiseArgumentError(
        Interpreter.buildArgumentErrorMsg(2, "out of range"),
      );
```